test/common/alerts_exist.go

```diff
@@ -35,6 +35,7 @@ var (
 	fileAdditional alertsTestFileStatus = "File found but not expected"
 	fileExists     alertsTestFileStatus = "File found with missing or unexpected rules"
 	fileCorrect    alertsTestFileStatus = "File found with all alerts present"
+	NamespacePrefixObservability = "redhat-rhoam-observability-"
```

Comment:
@valerymo would it be better to follow same approach as we currently do and push the namespace to test/common/types - where we could use existing prefix + observability and call it directly from there?